flang/lib/Lower/OpenMP.cpp

```diff
@@ -306,6 +306,35 @@ genOMP(Fortran::lower::AbstractConverter &converter,
   }
 }
 
+static mlir::omp::ClauseScheduleModifier
+translateModifier(const Fortran::parser::OmpScheduleModifierType &m) {
+  switch (m.v) {
+  case Fortran::parser::OmpScheduleModifierType::ModType::Monotonic:
+    return mlir::omp::ClauseScheduleModifier::Monotonic;
+  case Fortran::parser::OmpScheduleModifierType::ModType::Nonmonotonic:
+    return mlir::omp::ClauseScheduleModifier::Nonmonotonic;
+  default:
```

Comment:
Just an FYI. At one time, flang was using `-Wswitch` as an error to handle some unreachable cases at compile time.